### PR TITLE
review frontend

### PR DESCRIPTION
**Pull request by cdreamer1**

### CODE REVIEW

File: frontend/src/app/settings/profile/page.tsx [hunk 1]
```diff
@@ -98,13 +98,10 @@ const ProfileEditPage: React.FC = () => {
...
                         </div>
-
                     </div>
-
                     <div style={{ backgroundImage: 'linear-gradient(to right,rgb(190 191 195), rgb(101 101 101))' }} className="p-2 w-full mt-5 cursor-pointer hover:text-black">
                         <span style={{ fontWeight: '500' }}>Profile</span>
                     </div>
-
                     {
                         Items.map((item, index) => {
```
Comment (hi-zeus) [inline frontend/src/app/settings/profile/page.tsx:106]: Use camel case.

File: frontend/src/app/workSpace/@details/[...moduleName]/components/module-details-modal/module-details-modal.tsx
Comment (hi-zeus) [inline frontend/src/app/workSpace/@details/[...moduleName]/components/module-details-modal/module-details-modal.tsx:1]: - Fix the file structure (using Atomic design structure)
- Move the components files in to components directory. (atoms, molecules, organisms, templates)

File: frontend/src/app/workSpace/CustomNode.tsx
Comment (hi-zeus) [inline frontend/src/app/workSpace/CustomNode.tsx:1]: - Fix the file structure (using Atomic design structure)
- Move the components files in to components directory. (atoms, molecules, organisms, templates)

File: frontend/src/app/workSpace/components/module-item/index.tsx
Comment (hi-zeus) [inline frontend/src/app/workSpace/components/module-item/index.tsx:1]: - Fix the file structure (using Atomic design structure)
- Move the components files in to components directory. (atoms, molecules, organisms, templates)

File: frontend/src/app/workSpace/components/module-tile/module-tile.tsx
Comment (hi-zeus) [inline frontend/src/app/workSpace/components/module-tile/module-tile.tsx:1]: - Fix the file structure (using Atomic design structure)
- Move the components files in to components directory. (atoms, molecules, organisms, templates)

File: frontend/src/app/workSpace/components/search-bar.tsx
Comment (hi-zeus) [inline frontend/src/app/workSpace/components/search-bar.tsx:1]: - Fix the file structure (using Atomic design structure)
- Move the components files in to components directory. (atoms, molecules, organisms, templates)

File: frontend/src/components/substrate.component.tsx
Comment (hi-zeus) [inline frontend/src/components/substrate.component.tsx:1]: Move in to directory.

**Review by hi-zeus — CHANGES_REQUESTED** [review]
@cdreamer1 
- It would be better if you added types for hooks.
- Remove unnecessary code completely. (not comment)

Thanks

**Comment by hi-zeus** [comment]
@Tunki1201  
If you merge code without checking the modifications after a code review, a review is not necessary.

**Comment by Tunki1201** [comment]
Yes, I am.
Did not you also review the code?

File: frontend/src/app/settings/profile/page.tsx (hunk 1 above)
Comment (cdreamer1) [reply frontend/src/app/settings/profile/page.tsx:106]: changed Items to items.

File: frontend/src/components/substrate.component.tsx
Comment (cdreamer1) [reply frontend/src/components/substrate.component.tsx:1]: Moved this component into components/atoms/button directory.

File: frontend/src/app/workSpace/components/module-item/index.tsx
Comment (cdreamer1) [reply frontend/src/app/workSpace/components/module-item/index.tsx:1]: Moved this component into the workspace/components/molecules/ directory.

File: frontend/src/app/workSpace/components/module-tile/module-tile.tsx
Comment (cdreamer1) [reply frontend/src/app/workSpace/components/module-tile/module-tile.tsx:1]: Moved this component into the workspace/components/molecules/ directory.

File: frontend/src/app/workSpace/CustomNode.tsx
Comment (cdreamer1) [reply frontend/src/app/workSpace/CustomNode.tsx:1]: Moved this component into workspace/components/organisms/ directory.

**Comment by hi-zeus** [comment]
@Tunki1201 
After review,  you have to check the fix result.
After confirm, then you can merge it.

**Comment by Tunki1201** [comment]
ok, got it

**Pull request merged**